dnf-behave-tests/dnf/steps/server.py

```diff
@@ -107,3 +107,10 @@ def step_http_log_contains(context):
     for e in expected:
         if e not in found:
             raise AssertionError('HTTP log does not contain "%s": %s' % (e, "\n" + "\n".join(found) + "\n"))
+
+
+@behave.step("HTTP server GET {url} response headers are")
+def step_http_server_get_response_headers(context, url):
+    headers = [tuple(context.text.split('=', 1))]
```

Comment:
In general the `context.text` can be multiline with one header per line, can you please iterate the lines and add them to the dict one by one?